app/routes/company/components/CompaniesPage.tsx

```diff
@@ -84,20 +89,48 @@ const CompanyList = ({ companies = [] }: CompanyListProps) => (
   </div>
 );
 
+export const companiesDefaultQuery = {
+  search: '',
+  showInactive: 'false' as 'true' | 'false',
+};
+
+const filterCompanies = (
+  companies: ListCompany[],
+  query: typeof companiesDefaultQuery,
+) => {
+  return companies.filter((company) => {
+    const searchMatch =
+      !query.search ||
+      company.name.toLowerCase().includes(query.search.toLowerCase()) ||
+      company.description?.toLowerCase().includes(query.search.toLowerCase());
+
+    const activeMatch = !query.showInactive || company.active !== false;
+
+    return searchMatch && activeMatch;
+  });
+};
```

Comment:
Isn't doing this in the frontend a bit of a bad idea? Considering that the companies page is paginated..

Reply:
Yes indeed - I guess this is more like a mvp prototype .. I'll get around and see what I can do

Reply:
Implemented! https://github.com/webkom/lego/pull/3706